ht_exclude_bib.txt

```diff
@@ -0,0 +1,129 @@
+b6863323
```

Comment:
This is fine for now. One thing that I think would be good long-term is to have this be a tab-delimited file, where we can enter a brief reason why it's excluded in the 2nd column. Thoughts?